flink-ml-lib/src/test/java/org/apache/flink/ml/clustering/AgglomerativeClusteringTest.java

```diff
@@ -205,19 +206,19 @@ public void testTransform() throws Exception {
         Table[] outputs;
         AgglomerativeClustering agglomerativeClustering =
                 new AgglomerativeClustering()
-                        .setLinkage(AgglomerativeClusteringParams.LINKAGE_AVERAGE)
+                        .setLinkage(AgglomerativeClusteringParams.LINKAGE_WARD)
```

Comment:
It is actually a typo here because the expected results are name as `...WARD...`. 

I double checked with sklearn's results and they are consistent.